code/modules/clothing/gloves/color.dm

```diff
@@ -130,21 +130,39 @@
 
 /obj/item/clothing/gloves/color/latex
 	name = "latex gloves"
-	desc = "Cheap sterile gloves made from latex."
+	desc = "Cheap sterile gloves made from latex. Transfers minor paramedic knowledge to the user via budget nanochips."
 	icon_state = "latex"
 	item_state = "lgloves"
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
 	transfer_prints = TRUE
 	resistance_flags = NONE
 
+/obj/item/clothing/gloves/color/latex/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && slot == SLOT_GLOVES)

```

Comment:
It doesn't matter if they're human or not, only humans can firecarry.

Reply:
pretend I said this for all the copypasta (although that should be removed)

Reply:
i think i fixed the pasta maybve